### DISCUSSION TITLE
The ability to download files from urls

### DISCUSSION THREAD

**Discussion by ikfir** [pelican-dev/panel#671] [Ideas]
I would like to see button to upload from url in files
also it will be nice to have `hangar paper`/`spigot`/`modrinth` download from you will be able to download mods/plugins map etc...

p.s I don't mean just minecraft website I mean minecraft websites as easier to download mods from them but I want button to upload from url to just have to give the server http/https and it will be able to download from there

**Comment by gOOvER** [comment]
Nothing for Pelican in my eye's becaus not everyone plays minecraft. Would be an option for a plugin

**Comment by notAreYouScared** [comment]
In the new client area you can input a url for wings to download to your servers data folder.

**Comment by Boy132** [comment]
> I would like to see button to upload from url in files

This will be added in the new client area.

> also it will be nice to have hangar paper/spigot/modrinth download from you will be able to download mods/plugins map etc...

This will not be added to the panel itself but is a good idea for a plugin!
